src/components/VariantSelector/compact/index.ts

```diff
@@ -0,0 +1,157 @@
+import { fetchProduct } from "@/shopify/graphql/fetchProduct"
```

Comment:
drop all selectedOptions usage from this file and use the select element to get the selected variant